syne_tune/optimizer/schedulers/searchers/regularized_evolution.py

```diff
@@ -0,0 +1,138 @@
+import copy
+
+from collections import deque
+from typing import Optional, Dict, List
+from dataclasses import dataclass
+
+from syne_tune.optimizer.schedulers.searchers import SearcherWithRandomSeed
+from syne_tune.search_space import Domain, Categorical, Float, Integer
+
+
+@dataclass
+class PopulationElement:
+    score: int = 0
+    config: dict = None
+
+
+class RegularizedEvolution(SearcherWithRandomSeed):
```

Comment:
Thanks!

Reply:
Do you also want to link it into searcher_factory, scheduler_factory, or do this latewr?

Reply:
let's do this in a separate PR